packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -78,7 +78,7 @@ export function makeVatKeeper(
   function setSourceAndOptions(source, options) {
     // take care with API change
     assert(options.managerType, X`vat options missing managerType`);
-    assert.typeof(source, 'object');
+    assert(source);
```

Comment:
This seems like a lossy change.  If `source` is, say, a string the check will now pass even though it should not.

Reply:
There's already another assert on the next line that would fail on strings and such: `assert(source.bundle || source.bundleName);`.